src/adapter/http/src/data/mod.rs

```diff
@@ -7,12 +7,15 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+mod account_handler;
```

Comment:
I think all three of these new handlers don't belong in `data` group. I created it for raw data queries and commitments. Let's create some other group, perhaps called `general`?

Reply:
Yeah, agree, added